shared/styles/index.native.js

```diff
@@ -104,6 +105,9 @@ const globalStyles = {
 
 const navBarHeight = OS === OS_ANDROID ? 60 : NavigatorNavigationBarStyles.General.TotalNavHeight
 const tabBarHeight = 48
+// FIXME: StatusBar.currentHeight returns undefined on iOS in RN 0.34
+const statusBarHeight = OS === OS_ANDROID ? StatusBar.currentHeight : 20
```

Comment:
I think it does if you turn on translucent status bar, which I added to `index.native` (since we'll want this to implement the colod bleed in the designs).